### PR TITLE
Add build tools at version v0.0.55

### PR DESCRIPTION
**Pull request by TheoLeanse**
@commuterjoy @matthew-andrews @keirog 

Is this the right way?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,44 +1,6 @@
-TEST_HOST := "ft-prodsel-branch-${CIRCLE_BUILD_NUM}"
```
Comment (matthew-andrews) [inline Makefile]: you'll need to keep this line

File: Makefile [hunk 2]
```diff
@@ -1,44 +1,6 @@
...
-	nbt deploy --skip-logging
-
-provision:
-	nbt provision ${TEST_HOST}
-	nbt configure ft-next-stream-page ${TEST_HOST} --overrides "NODE_ENV=branch"
-	nbt deploy-hashed-assets
-	nbt deploy ${TEST_HOST} --skip-enable-preboot --skip-logging
-
-tidy:
-	nbt destroy ${TEST_HOST}
```
Comment (matthew-andrews) [inline Makefile]: for now you'll need to make `run`, `deploy`, `provision` and `tidy` … that stuff hasn't quite made it into `n.Makefile` yet

**Comment by matthew-andrews** [comment]
almost!

**Comment by TheoLeanse** [comment]
This?

File: Makefile [hunk 3]
```diff
@@ -1,25 +1,6 @@
...
-
-install:
-	npm install origami-build-tools
-	origami-build-tools install
+include n.Makefile
 
-watch:
-	nbt build --dev --watch
-
-build:
```
Comment (matthew-andrews) [inline Makefile]: ahh also `watch`, `build`, and `build-production` aren't quite migrated either sorry :grin:

**Comment by TheoLeanse** [comment]
@matthew-andrews @commuterjoy apologies for all the spam but I think this may now be ok to merge - OK?

**Comment by matthew-andrews** [comment]
+1

**Pull request merged**